eng/packages/http-client-csharp/generator/Azure.Generator/src/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.TypeSpec.Generator;
+
+namespace Azure.Generator
+{
+    internal static class ConfigurationExtensions
+    {
+        public static bool UseModelNamespace(this Configuration config) =>
+            UseModelNamespace(config.AdditionalConfigurationOptions);
+
+        internal static bool UseModelNamespace(IReadOnlyDictionary<string, BinaryData> options)
+        {
+            return !options.TryGetValue("model-namespace", out var value)
```

Comment:
I thought dataplane wanted this to default to false and mgmt plane wants it to default to true.  @christothes do you know if my memory is correct here and do we have it documented anywhere?

Reply:
It is defaulted to true in autorest, so we need to maintain backcompat.

Reply:
But if all dataplane libraries are setting it to false, we can flip this.